### DISCUSSION TITLE
Need help with deploying!

### DISCUSSION THREAD

**Discussion by matt-cha** [vercel/community#1992] [Help]
### Summary

Hello!

I'm having trouble deploying my Next.js app on Vercel. I'm currently getting the error "Build Failed
No Next.js version could be detected in your project. Make sure `"next"` is installed in "dependencies" or "devDependencies""

in my package.json file, it is in the root of my project and as a dependency, next 13.2.4 is installed. Please let me know if there's any other info you need! Not sure what's wrong here seems it should be a simple fix

### Additional information
![Screen Shot 2023-04-05 at 6 10 13 PM](https://user-images.githubusercontent.com/113474295/230247681-4024e996-c052-45e4-a2be-86fe4e8944c0.png)
![Screen Shot 2023-04-05 at 6 10 32 PM](https://user-images.githubusercontent.com/113474295/230247717-37948d26-8087-4ffa-b940-2c2c40f4aff7.png)

_No response_

**Comment by Abhisha728Kumar** [comment]
I also got the same error while deploying my first next js app to vercel
There are few points that you can follow:
1. Try to run your Development server and check if you get and error modal thrown by next js.
2. Try to run command npm run lint in your terminal and check if any error logs in your terminal.
3. Try to run command npm run build in your terminal amd check if something start malfunctioning or show and error.
THESE FEW POINTS MAY HELP YOU TO RESOLVE YOUR PROBLEM.

**Reply by matt-cha** [reply]
Hi! I tried all 3 of these and there are warnings about the <img> tag I'm using but no errors! Still does not deploy unfortunately saying the same error 
<img width="577" alt="Screen Shot 2023-04-05 at 8 56 41 PM" src="https://user-images.githubusercontent.com/113474295/230268357-2ecdc395-efce-440b-868c-ce938b1f0d26.png">
<img width="540" alt="Screen Shot 2023-04-05 at 8 56 58 PM" src="https://user-images.githubusercontent.com/113474295/230268386-bd33e96a-06d9-426f-a2da-50b993263d73.png">
<img width="781" alt="Screen Shot 2023-04-05 at 8 58 14 PM" src="https://user-images.githubusercontent.com/113474295/230268585-225ca8dc-49e1-45d8-b1cf-4c632d702608.png">

**Comment by Abhisha728Kumar** [comment]
You can try to delete the node modules and installing again, it may show some effects. If nothing changes same error comes the you can send me the screenshot of your package.json file and the error  which comes after all.

**Comment by avmentzer** [comment]
Same here:

`<html>
<body>


Warning: Could not identify Next.js version, ensure it is defined as a project dependency.
--
16:50:35.574 | Error: No Next.js version could be detected in your project. Make sure `"next"` is installed in "dependencies" or "devDependencies"
16:50:35.682 | NEXT_NO_VERSION: No Next.js version could be detected in your project. Make sure `"next"` is installed in "dependencies" or "devDependencies"


</body>
</html>`


package.json:

`"scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint"
  },
  "dependencies": {
    "@geoapify/geocoder-autocomplete": "^1.5.1",
    "@geoapify/react-geocoder-autocomplete": "^1.5.0",
    "@prisma/client": "^4.12.0",
    "@types/bcryptjs": "^2.4.2",
    "@types/jsonwebtoken": "^9.0.2",
    "@types/node": "18.15.11",
    "@types/react": "18.0.35",
    "@types/react-dom": "18.0.11",
    "bcrypt-ts": "^3.0.1",
    "cookies": "^0.8.0",
    "cookies-next": "^2.1.1",
    "daisyui": "^2.51.6",
    "iron-session": "^6.3.1",
    "jsonwebtoken": "^9.0.0",
    "leaflet": "^1.9.4",
    "leaflet-geosearch": "^3.8.0",
    "next": "^13.4.4",
    "nextjs": "^0.0.3",
    "react": "^18.2.0",
    "react-cookie": "^4.1.1",
    "react-dom": "^18.2.0",
    "react-geolocated": "^4.1.0",
    "react-leaflet": "^4.2.1",
    "scrypt-js": "^3.0.1",
    "typescript": "5.0.4"
  },
  "devDependencies": {
    "@types/bcrypt": "^5.0.0",
    "@types/cookie": "^0.5.1",
    "@types/leaflet": "^1.9.3",
    "add": "^2.0.6",
    "autoprefixer": "^10.4.14",
    "cookie": "^0.5.0",
    "next": "^13.4.4",
    "postcss": "^8.4.22",
    "prisma": "^4.12.0",
    "tailwindcss": "^3.3.1"
  }
}`
package.json in .next:

`{"type": "commonjs"}`
